sourcemash/api/items.py

```diff
@@ -98,6 +103,7 @@ def put(self, id):
         # Toggle saved-for-later status (aka bookmarked)
         if args.saved != None:
             user_item.saved = args.saved
+            user_item.last_modified = datetime.utcnow()
```

Comment:
We can probably just call this once since something needs to have changed if the API call was even made. Then we won't need it in the args.vote check too.